build.gradle.kts

```diff
@@ -111,4 +111,8 @@ tasks {
         // https://plugins.jetbrains.com/docs/intellij/deployment.html#specifying-a-release-channel
         channels.set(listOf(properties("pluginVersion").split('-').getOrElse(1) { "default" }.split('.').first()))
     }
+
+    runPluginVerifier {
+        ideVersions.set(listOf("IIC-222.3739.24", "IIC-221.6008.8"))
```

Comment:
Where are we getting these from? It might be good to add a comment for the next time we might need to update.

Reply:
I used this: https://github.com/bufbuild/intellij-buf/pull/33.

I can add it